nservicebus/upgrades/5to6.md

```diff
@@ -353,6 +353,11 @@ A new [Message Routing Feature](/nservicebus/messaging/routing.md) has been adde
 
 Previous versions of NServiceBus used `TransportMessage` as a generic container both for outgoing and incoming messages. For a better separation of concerns that class has been split into `IncomingMessage` and `OutgoingMessage`. So all code paths related to outgoing messages will use `OutgoingMessage` and all code paths related to incoming messages will use `IncomingMessages`. The class `TransportMessage` has been deprecated entirely. Here are a few common scenarios related to `TransportMessage` and how they can be addressed with either `IncomingMessage` or `OutgoingMessage`.
 
+## Deprecated Address
+
+Previous versions of NServiceBus forced usage of `Address` class as a representation of an address in all transports despite the fact that the form *queue@host* did not match most of the transports. In Version 6 the transport address is represented as an opaque `string` and NServiceBus core no longer makes any assumptions on its form.
```

Comment:
Rather than "forced usage of `Address` class" can we just say 

> Version 5 of NServiceBus represents addresses with an `Address` class. The `Address` class maintains addresses in the *queue@host* format. This format was originally developed for the MSMQ transport but does not meet the needs of other transports. In Version 6, addresses are represented as opaque strings.